Milvus.Client/MilvusCollection.Entity.cs

```diff
@@ -554,7 +564,7 @@ public Task<FlushResult> FlushAsync(CancellationToken cancellationToken = defaul
                     Value = Math.Min(batchSize, leftItemsCount).ToString(CultureInfo.InvariantCulture)
                 });
 
-            var nextExpression = pkField.DataType == DataType.VarChar
+            string nextExpression = pkField.DataType == DataType.VarChar
```

Comment:
Please do the same switch here as above, with an exception for unsupported types etc.

Reply:
added